src/main/java/com/redhat/devtools/lsp4ij/launching/templates/LanguageServerTemplateManager.java

```diff
@@ -66,4 +76,123 @@ static Reader loadTemplateReader(@NotNull String path) {
         var is = LanguageServerTemplateManager.class.getClassLoader().getResourceAsStream(TEMPLATES_DIR + "/" + path);
         return is !=null? new InputStreamReader(new BufferedInputStream(is)) : null;
     }
-}
+
+    @Nullable
+    public LanguageServerTemplate importLsTemplate(@NotNull VirtualFile templateFolder) {

```

Comment:
I think we should have 2 import methods:

 * one which import a template from a given directory like you did
 * one which import a template from a zip file

Reply:
Would it be fine to create a new issue for importing a zip file?

Reply:
Could you create an issue please.

Reply:
Sure thing :+1: